llvm/test/CodeGen/AMDGPU/amdgpu-sw-lower-lds-non-kernel-declaration.ll

```diff
@@ -0,0 +1,97 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-globals all --version 5
+; RUN: opt < %s -passes=amdgpu-sw-lower-lds -S -mtriple=amdgcn-amd-amdhsa | FileCheck %s
+@lds = external addrspace(3) global [5 x i8], align 8
+declare void @kernel_declaration() sanitize_address
```

Comment:
Don't call this kernel_declaration, it's not a kernel

Reply:
updated in latest commit.